setup.py

```diff
@@ -496,6 +498,69 @@ def _read_requirements(filename: str) -> List[str]:
     return requirements
 
 
+def repackage_wheel(package_data: Dict[str, List[str]],
+                    wheel_location: str) -> None:
+    """Extracts libraries and other files from an existing wheel."""
+    assert _is_cuda(), "VLLM_USE_PRECOMPILED is only supported for CUDA builds"
+
+    import zipfile
+
+    if os.path.isfile(wheel_location):
+        wheel_path = wheel_location
+        print(f"Using existing wheel={wheel_path}")
+    else:
+        # Download the wheel from a given URL, assume
```

Comment:
since we are just downloading a wheel file here, `pip` cannot do anything. directly fetch the url.